sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -5899,4 +5899,22 @@ ALTER TABLE person_history ADD COLUMN armedforcesrelationtype varchar(255);
 
 INSERT INTO schema_version (version_number, comment) VALUES (283, 'Add column armedforcesrelationtype #3418');
 
--- *** Insert new sql commands BEFORE this line ***
+-- 2020-11-17 Manually send SMS #3253
+CREATE TABLE manualmessagelog
```

Comment:
primary key and sys_period are missing

Reply:
ok
i'm not sure the sys_period is needed in this case, why would we need it ?

Reply:
You're right, we don't need a history table so should not be necessary